lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -1464,8 +1471,13 @@ Error Caffe2ModelLoader::loadNetwork(caffe2::NetDef &net) {
     auto &outputName = net.external_output(i);
     NodeValue r;
     ASSIGN_VALUE_OR_RETURN_ERR(r, getNodeValueByName(outputName));
```

Comment:
I don't understand something here: if nodes and placeholders share a namespace, wouldn't this `getNodeValueByName` fail to find the desired node, since it's probably been re-named something like `output__1`?

Reply:
Good question. It clearly works for e.g resnet, but your explanation sounds right. I'll look into it.

Reply:
OK, so the way this works is `getNodeValueByName` has its own map (ie not the Function/Module) which claiming a name in the Module doesn't affect. This is good because we're looking up illegal names directly, and double good because we write into that name without checking if the name is already taken. TLDR: nodeValueByName uses different names than the Nodes themselves.